diff -ur avant-window-navigator-0.2.6.orig/awn-manager/awnManager.py.in avant-window-navigator-0.2.6/awn-manager/awnManager.py.in
--- avant-window-navigator-0.2.6.orig/awn-manager/awnManager.py.in	2008-04-21 12:56:13.000000000 +0200
+++ avant-window-navigator-0.2.6/awn-manager/awnManager.py.in	2008-04-21 12:58:27.000000000 +0200
@@ -22,7 +22,7 @@
 
 import sys, os, os.path, time
 
-SITE_PKG = '@PY_SITE_PKG@'
+SITE_PKG = '/usr/lib/python'+sys.version[0:3]+'/site-packages'
 AWN_MANAGER_DIR = os.path.join('@PKGDATADIR@', 'awn-manager')
 sys.path = [AWN_MANAGER_DIR] + sys.path
 if SITE_PKG not in sys.path:
Seulement dans avant-window-navigator-0.2.6/awn-manager: awnManager.py.in~
