--- awn-manager/awnManager.py.in	2008-05-20 11:25:44.976064040 +0200
+++ awn-manager/awnManager.py.in	2008-05-20 11:25:24.497177304 +0200
@@ -87,7 +87,7 @@
         self.gtk_screen = self.window.get_screen()
         colormap = self.gtk_screen.get_rgba_colormap()
         if colormap == None:
-            colormap = screen.get_rgb_colormap()
+            colormap = self.gtk_screen.get_rgb_colormap()
         gtk.widget_set_default_colormap(colormap)
 
         self.notebook = self.wTree.get_widget("panelCategory")
