--- avant-window-navigator-0.4.0.orig/src/Makefile.in
+++ avant-window-navigator-0.4.0/src/Makefile.in
@@ -310,6 +310,7 @@ AM_CPPFLAGS = \
 avant_window_navigator_LDADD = \
 	$(DOCK_LIBS)				\
 	$(top_builddir)/libawn/libawn.la	\
+	$(AWN_LIBS) \
 	$(NULL)
 
 avant_window_navigator_SOURCES = \
--- avant-window-navigator-0.4.0.orig/src/Makefile.am
+++ avant-window-navigator-0.4.0/src/Makefile.am
@@ -18,6 +18,7 @@ bin_PROGRAMS = avant-window-navigator
 avant_window_navigator_LDADD =			\
 	$(DOCK_LIBS)				\
 	$(top_builddir)/libawn/libawn.la	\
+	$(AWN_LIBS) \
 	$(NULL)
 
 avant_window_navigator_SOURCES =	\
--- avant-window-navigator-0.4.0.orig/tests/Makefile.in
+++ avant-window-navigator-0.4.0/tests/Makefile.in
@@ -58,7 +58,7 @@ mkinstalldirs = $(SHELL) $(top_srcdir)/m
 CONFIG_HEADER = $(top_builddir)/config.h
 CONFIG_CLEAN_FILES =
 CONFIG_CLEAN_VPATH_FILES =
-@HAVE_VALA_TRUE@am__EXEEXT_1 = test-vala-awn-dialog$(EXEEXT)
+#@HAVE_VALA_TRUE@am__EXEEXT_1 = test-vala-awn-dialog$(EXEEXT)
 PROGRAMS = $(noinst_PROGRAMS)
 am_test_applet_simple_OBJECTS = test-applet-simple.$(OBJEXT)
 test_applet_simple_OBJECTS = $(am_test_applet_simple_OBJECTS)
@@ -315,13 +315,13 @@ AM_CPPFLAGS = $(STANDARD_CPPFLAGS) $(DIS
 AM_CFLAGS = $(WARNING_FLAGS)
 INCLUDES = $(AWN_CFLAGS) -I$(top_srcdir)
 test_applet_simple_SOURCES = test-applet-simple.c
-test_applet_simple_LDADD = $(top_builddir)/libawn/libawn.la
+test_applet_simple_LDADD = $(top_builddir)/libawn/libawn.la $(AWN_LIBS)
 test_awn_effects_SOURCES = test-awn-effects.c
-test_awn_effects_LDADD = $(top_builddir)/libawn/libawn.la
+test_awn_effects_LDADD = $(top_builddir)/libawn/libawn.la $(AWN_LIBS)
 test_awn_icon_SOURCES = test-awn-icon.c
-test_awn_icon_LDADD = $(top_builddir)/libawn/libawn.la
+test_awn_icon_LDADD = $(top_builddir)/libawn/libawn.la $(AWN_LIBS)
 test_awn_icon_box_SOURCES = test-awn-icon-box.c
-test_awn_icon_box_LDADD = $(top_builddir)/libawn/libawn.la
+test_awn_icon_box_LDADD = $(top_builddir)/libawn/libawn.la $(AWN_LIBS)
 test_taskmanager_SOURCES = test-taskmanager.c
 test_taskmanager_LDADD = \
 	$(AWN_LIBS) \
@@ -329,7 +329,7 @@ test_taskmanager_LDADD = \
 	$(NULL)
 
 test_themed_icon_SOURCES = test-themed-icon.c
-test_themed_icon_LDADD = $(top_builddir)/libawn/libawn.la
+test_themed_icon_LDADD = $(top_builddir)/libawn/libawn.la $(AWN_LIBS)
 EXTRA_DIST = test-awn-dialog.py test-awn-tooltip.py test-effects.py \
 	test-effects-scaling.py test-overlays.py \
 	test-taskmanager-dnd.py test-taskmanager-windows.py \
--- avant-window-navigator-0.4.0.orig/tests/Makefile.am
+++ avant-window-navigator-0.4.0/tests/Makefile.am
@@ -13,16 +13,16 @@ AM_CFLAGS = $(WARNING_FLAGS)
 INCLUDES = $(AWN_CFLAGS) -I$(top_srcdir)
 
 test_applet_simple_SOURCES = test-applet-simple.c
-test_applet_simple_LDADD = $(top_builddir)/libawn/libawn.la
+test_applet_simple_LDADD = $(top_builddir)/libawn/libawn.la $(AWN_LIBS)
 
 test_awn_effects_SOURCES = test-awn-effects.c
-test_awn_effects_LDADD = $(top_builddir)/libawn/libawn.la
+test_awn_effects_LDADD = $(top_builddir)/libawn/libawn.la $(AWN_LIBS)
 
 test_awn_icon_SOURCES = test-awn-icon.c
-test_awn_icon_LDADD = $(top_builddir)/libawn/libawn.la
+test_awn_icon_LDADD = $(top_builddir)/libawn/libawn.la $(AWN_LIBS)
 
 test_awn_icon_box_SOURCES = test-awn-icon-box.c
-test_awn_icon_box_LDADD = $(top_builddir)/libawn/libawn.la
+test_awn_icon_box_LDADD = $(top_builddir)/libawn/libawn.la $(AWN_LIBS)
 
 test_taskmanager_SOURCES = test-taskmanager.c
 test_taskmanager_LDADD = \
@@ -31,7 +31,7 @@ test_taskmanager_LDADD = \
 	$(NULL)
 
 test_themed_icon_SOURCES = test-themed-icon.c
-test_themed_icon_LDADD = $(top_builddir)/libawn/libawn.la
+test_themed_icon_LDADD = $(top_builddir)/libawn/libawn.la $(AWN_LIBS)
 
 EXTRA_DIST = 	test-awn-dialog.py 	\
 		test-awn-tooltip.py	\
@@ -42,7 +42,7 @@ EXTRA_DIST = 	test-awn-dialog.py 	\
 		test-taskmanager-windows.py
 
 if HAVE_VALA
-noinst_PROGRAMS += test-vala-awn-dialog
+#noinst_PROGRAMS += test-vala-awn-dialog
 
 VALA_FILES = \
 	test-vala-awn-dialog.vala \
